src/test/java/redis/clients/jedis/tests/JedisClusterTest.java

```diff
@@ -510,24 +566,35 @@ public void testCloseable() throws IOException {
     JedisCluster jc = null;
     try {
       jc = new JedisCluster(jedisClusterNode, DEFAULT_TIMEOUT, DEFAULT_TIMEOUT,
-          DEFAULT_REDIRECTIONS, "cluster", DEFAULT_CONFIG);
+          DEFAULT_REDIRECTIONS, "cluster", DEFAULT_POOL_CONFIG);
       jc.set("51", "foo");
     } finally {
       if (jc != null) {
         jc.close();
       }
     }
 
-    Iterator<JedisPool> poolIterator = jc.getClusterNodes().values().iterator();
-    while (poolIterator.hasNext()) {
-      JedisPool pool = poolIterator.next();
-      try {
-        pool.getResource();
-        fail("JedisCluster's internal pools should be already destroyed");
-      } catch (JedisConnectionException e) {
-        // ok to go...
-      }
-    }
+//    Iterator<JedisPool> poolIterator = jc.getClusterNodes().values().iterator();
+//    while (poolIterator.hasNext()) {
+//      JedisPool pool = poolIterator.next();
+//      try {
+//        pool.getResource();
+//        fail("JedisCluster's internal pools should be already destroyed");
+//      } catch (JedisConnectionException e) {
+//        // ok to go...
+//      }
+//    }
```

Comment:
I think you forgot to uncomment this?

Reply:
Nope.

I debugged and found that `jc.getClusterNodes().values()` is actually empty, even without my changes. So above codes are meaningless.

Also check that following `assertEquals(0, jc.getClusterNodes().size());` is passing.

Reply:
If that part is useless let's delete it then, commenting it out leave unanswered questions